springfox-petstore-webflux/src/main/java/springfox/petstore/webflux/controller/UserController.java

```diff
@@ -65,7 +65,7 @@
   @ResponseBody
   @ApiOperation(value = "Creates list of users with given input array")
   public Mono<ResponseEntity<User>> createUsersWithArrayInput(@ApiParam(value = "List of user object", required = true)
-                                                        User[] users) {
+                                                        @RequestBody User[] users) {
```

Comment:
Isnt this incorrect? You're fixing the existing tests by adding this request body. In essence you're not preserving the existing behavior